project/Dependencies.scala

```diff
@@ -87,31 +87,27 @@ object Dependencies {
   val ehcache = "org.ehcache" % "ehcache" % "3.10.8"
 
   // other
-  val diff           = "com.sksamuel.diff" % "diff"                % "1.1.11"
-  val gwtServlet     = "com.google.gwt"    % "gwt-servlet"         % "2.10.0"
-  val icu4j          = "com.ibm.icu"       % "icu4j"               % "75.1"
-  val jakartaJSON    = "org.glassfish"     % "jakarta.json"        % "2.0.1"
-  val rdf4jClient    = "org.eclipse.rdf4j" % "rdf4j-client"        % Rdf4jVersion
-  val rdf4jShacl     = "org.eclipse.rdf4j" % "rdf4j-shacl"         % Rdf4jVersion
-  val rdf4jSparql    = "org.eclipse.rdf4j" % "rdf4j-sparqlbuilder" % Rdf4jVersion
-  val saxonHE        = "net.sf.saxon"      % "Saxon-HE"            % "12.4"
-  val scalaGraph     = "org.scala-graph"  %% "graph-core"          % "2.0.1" // Should be Scala 3 compatible
-  val titaniumJSONLD = "com.apicatalog"    % "titanium-json-ld"    % "1.4.0"
-  val xmlunitCore    = "org.xmlunit"       % "xmlunit-core"        % "2.10.0"
+  val diff        = "com.sksamuel.diff" % "diff"                % "1.1.11"
+  val gwtServlet  = "com.google.gwt"    % "gwt-servlet"         % "2.10.0"
+  val icu4j       = "com.ibm.icu"       % "icu4j"               % "74.2"
+  val jakartaJSON = "org.glassfish"     % "jakarta.json"        % "2.0.1"
+  val rdf4jClient = "org.eclipse.rdf4j" % "rdf4j-client"        % Rdf4jVersion
+  val rdf4jShacl  = "org.eclipse.rdf4j" % "rdf4j-shacl"         % Rdf4jVersion
+  val rdf4jSparql = "org.eclipse.rdf4j" % "rdf4j-sparqlbuilder" % Rdf4jVersion
+  val saxonHE     = "net.sf.saxon"      % "Saxon-HE"            % "12.4"
+  val scalaGraph =
+    "org.scala-graph" %% "graph-core" % "2.0.1" cross CrossVersion.for3Use2_13 // Should be Scala 3 compatible
+  val titaniumJSONLD = "com.apicatalog" % "titanium-json-ld" % "1.4.0"
+  val xmlunitCore    = "org.xmlunit"    % "xmlunit-core"     % "2.10.0"
 
   // test
   val pekkoHttpTestkit   = "org.apache.pekko" %% "pekko-http-testkit"   % PekkoHttpVersion  // Scala 3 incompatible
   val pekkoStreamTestkit = "org.apache.pekko" %% "pekko-stream-testkit" % PekkoActorVersion // Scala 3 compatible
   val pekkoTestkit       = "org.apache.pekko" %% "pekko-testkit"        % PekkoActorVersion // Scala 3 compatible
   val scalaTest          = "org.scalatest"    %% "scalatest"            % "3.2.18"          // Scala 3 compatible
```

Comment:
suggestion (non-blocking): I think it would make sense to remove all the comments relating to Scala 3 compatibility

Reply:
https://github.com/dasch-swiss/dsp-api/pull/3231/commits/a768fcfd65aa4a31b806d73e2f60c341993ce0df